bug: `Get Mathlib Cache` fails

Sometimes workflow fails with error message such that:

```
▼ Run : Get Mathlib Cache
  : Get Mathlib Cache
  echo "::group::Mathlib Cache"
  lake exe cache get
  echo "::endgroup::"
  echo
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
▼ Mathlib Cache
  info: mdgen: updating repository '././.lake/packages/mdgen' to revision '6a6a412465e82572f6e30385592eac52ab055c83'
  info: mathlib: updating repository '././.lake/packages/mathlib' to revision '03092720f68254315c95a5bf2[49](https://github.com/lean-ja/tactic-cheatsheet/actions/runs/9530357340/job/26270125659#step:5:55)c5f2cdef717d6'
  error: external command 'git' exited with code 1
  Error: Process completed with exit code 1.
```

see: https://github.com/leanprover/lean-action/issues/62

--- a/action.yml
+++ b/action.yml
@@ -62,6 +62,8 @@ runs:
       uses: leanprover/lean-action@main
       with:
         test: false
+        # If the .lake directory remains, an error may occur when updating.
+        use-github-cache: false
 
     - name: Record outcome
       id: record-outcome

